src/components/SearchForm.js

```diff
@@ -1,10 +1,48 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
+import axios from "axios";
+import CharacterCard from "./CharacterCard";
 
 export default function SearchForm() {
- 
+  const [lookNames, setLookNames] = useState([]);
+  const [csearch, newsearch] = useState("");
+  useEffect(() => {
+    axios
+      .get("https://rickandmortyapi.com/api/character/")
+      .then(res => {
+        console.log(res.data.results);
+        const characters = res.data.results;
+        const results = characters.filter(character => {
+          return character.name.toLowerCase().includes(csearch.toLowerCase());
+        });
+        setLookNames(results);
+      })
+      .catch(error => {
+        console.log(error);
+      });
+  }, [csearch]);
+  const onSearchChange = event => {
+    newsearch(event.target.value);
+    console.log(newsearch);
+  };
+  let searchResult = "";
+  var submit = event => {
+    return (searchResult = event.target.value);
+  };
+
   return (
-    <section className="search-form">
-     // Add a search form here
+    <section>
+      <form onSubmit={event => submit(event)} className="tb">
```

Comment:
The event argument is passed implicitly on event listeners in React.
You do not have to explicitly pass the event argument.
This could also work in the code above `onSubmit={submit}`